app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comments = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author_id = @user.id
+    @comment.post_id = @post.id
```

Comment:
- [ ] Hi you did a nice job but there is an issue when trying to add a comment. This is because the variable `@user` has not been declared so it doesn't have an `id`. Ill recommend using the current user id to create the new comment i.e `@comment.author_id = current_user.id`
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/37901469/205458290-c1957741-b27b-4676-8268-6952a2adf4ce.png">